config/mempool.mk

```diff
@@ -14,6 +14,9 @@ num_cores ?= 256
 # Number of groups
 num_groups ?= 4
 
+# Number of cores per Mempool group
+num_cores_per_group ?= 64
```

Comment:
Can we infer this parameter from the other values, instead of adding a new parameter to the list (which adds a lot of work to the other flows)?